Source/DataAccessLayer/Utility.cst

```diff
@@ -553,7 +553,7 @@ namespace <%=DALNameSpace%>
             CommandText = command.CommandText;
             CommandTimeout = command.CommandTimeout;
             CommandType = command.CommandType.ToString();
-            ConnectionString = command.Connection.ConnectionString;
+            ConnectionString = command.Connection == null ? string.Empty : command.Connection.ConnectionString;

```

Comment:
Can we change this to `command.Connection != null ? command.Connection.ConnectionString : null;`

Reply:
I will make this change.  I have a number of additional changes to add to this commit as well.